### DISCUSSION TITLE
Recommendations on Android SDK Dockers images

### DISCUSSION THREAD

**Discussion by ericoporto** [cirruslabs/cirrus-ci-docs#1133] [Q&A] [answered]
Hi,

Google released a new Android Studio (Dolphin) and I am looking into updating everything Android related in our projects to be compatible with the latest versions of things.

I see the Android SDK Docker images of CirrusCi are not updated.

What is the recommendation here, should I move to having my own images fully (not depend on Cirrus CI docker images) for Android builds or there is still interest from Cirrus CI to keep updating the Android SDK images. 

Is there any deprecation schedule for these to recommend people to maintain their own?

Some advice here would be useful!

**Comment by fkorotkov** [comment, marked as answer]
Hey @ericoporto,

Generally speaking Cirrus CI allows to use any images and many users create their own custom images curated for their workflows either manually or via [Dockerfile as CI environment](https://cirrus-ci.org/guide/docker-builder-vm/#dockerfile-as-a-ci-environment) feature. This gives full control over the execution environment and allows to pre-install some specific packages if a generic image misses them (which speeds up builds since everything is pre-installed).

Having said so, if images from https://github.com/cirruslabs/docker-images-android/ worked for you out of the box, I've created a PR to move to Android 33 https://github.com/cirruslabs/docker-images-android/pull/47. PTAL!

**Reply by ericoporto** [reply]
Thanks, the prebuilt images from CirrusCi helps me because it's one less thing I need to figure out and maintain, so if they will continue to be available and updated I will keep using it. :)
